db/migrate/20200911204218_create_problems.rb

```diff
@@ -5,6 +5,8 @@ def change
 
       t.string :mode, default: "anarchy", null: false
       t.boolean :complete, default: false, null: false
+      t.integer :tick, default: 0, null: false
+      t.jsonb :settings, default: '{}', null: false
```

Comment:
Will probably change this migration in the future to have a reasonable set of defaults for each program so all these options don't just start as `nil`